### PR TITLE
#157188286 Product ratings and reviews

### PR DESCRIPTION
**Pull request by FaithAdekunle**
#### What does this PR do?
Integrates product ratings and reviews into app

#### Description of Task to be completed?

- Allow a user to rate and review a product
- Allow user to overwrite his/her rating and review of a product
- Allow user and admin to see average rating of product
- Allow user and admin to read reviews of product

#### How should this be manually tested?

- npm run git clone `https://github.com/andela/nazgul-rc.git` to clone the repository
- navigate to the nazgul-rc folder
- run `curl https://install.meteor.com/ | sh`
- run `npm install -g reaction-cli`
- install dependencies with`meteor npm install`
- run `reaction` to start up server
- once app is running, navigate to the app using localhost:3000
- navigate to shop page
- click on a product
- see average rating for that product
- see reviews for that product
- rate and review the product

#### What are the relevant pivotal tracker stories?
#157188286

### CODE REVIEW

**Comment by morekoya** [comment]
This does not follow our PR naming convention.

**Comment by FaithAdekunle** [comment]
@andela-morekoya My apologies. I have renamed it. Thanks.

File: e2eTest/ratingsAndReviews.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+/* global module */
```
Comment (morekoya) [inline e2eTest/ratingsAndReviews.test.js:1]: Your test does allow check that unauthorized users cannot rate/review.
It also does not confirm that when a user submit a review, the review is saved/shows on the page

Comment (morekoya) [reply e2eTest/ratingsAndReviews.test.js:1]: Also the test should show that shop owners cannot rate their shops

Comment (FaithAdekunle) [reply e2eTest/ratingsAndReviews.test.js:1]: @andela-morekoya This feature is for product ratings and reviews. Do you mean the test should show that an admin cannot rate a product?

**Pull request merged**